sourabhkumar47/ResQFood
# added item detail screen fix #58

# Pull Request Template

## Description
Added the screen to display the food item details.

## Related Issue
 "Fixes #58 "

## Changes
New screen is added

## Testing Instructions
temporary added the button on the main screen, on clicking on it, this page is displayed. there was no parent component which can be used to navigate to this screen, thats why for temporary purpose I have added the button on the main screen, later remove it.

## Impact Analysis


- [ N ] Breaking change: This change introduces breaking modifications.
- [ N ] Non-breaking change: This change does not introduce breaking modifications.
- [ N ] Disrupts existing functionality: This change disrupts the functionality of existing features.
- [ N ] Does not disrupt existing functionality: This change does not disrupt the functionality of existing features.

## Screenshots (if applicable)
![Screenshot_20240527_174440](https://github.com/sourabhkumar47/ResQFood/assets/38074085/1c63050b-4ab1-4a76-bd3e-789b724bb140)
![Screenshot_20240527_174444](https://github.com/sourabhkumar47/ResQFood/assets/38074085/93d6e395-3ebf-41f1-87d6-822d6c5dc6ea)
![Screenshot_20240527_174453](https://github.com/sourabhkumar47/ResQFood/assets/38074085/0cfc2a52-4300-4d21-a009-d70917768589)

## Checklist
- [Y ] I have tested the changes locally and they work as expected.
- [ N ] I have updated the documentation if necessary.
- [ Y ] I have reviewed the code and ensured it follows project standards.
- [ Y  ] All tests (if applicable) pass successfully.
- [  N ] I have added myself to the list of contributors if this is a significant contribution.

## Additional Context
Its not accepting large size image.
